scripts/swupd-performance/swupdservicesetup.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/bash
+
+set -e
+
+echo "Setup staging directory"
+
+if [ -d "$HOME/staging" ]; then
+	echo "Staging directory already exists" || true
+else
+	dir=$HOME/staging
+	mkdir $HOME/staging
+
+	if [ $? -eq 0 ]; then
```

Comment:
This is probably a problem. Can you check with shellcheck? I think that if you rename the file to .bash make shellcheck should run automatically for this. if not, can you change the bats to include scripts/ directory?

Reply:
fixed this